samples/00-hello-webvr.html

```diff
@@ -12,7 +12,7 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
 
     <!--Chrome origin trial header-->
-    <meta http-equiv="origin-trial" data-feature="WebVR" data-expires="2017-03-08" content="AiTLRl2auC8Te9Yue7upVEdZUgdgEDgrDIB9A1TO+Gkaz4BXn5TX/gZ0/MibV3+epWy39bW1MjbE+9giEGuICAcAAABJeyJvcmlnaW4iOiJodHRwczovL3dlYnZyLmluZm86NDQzIiwiZmVhdHVyZSI6IldlYlZSIiwiZXhwaXJ5IjoxNDg4OTg2NjM0fQ==">
+    <meta http-equiv="origin-trial" data-feature="WebVR" data-expires="04/13/17" content="AtpEVUJTjLpU/IMdKA/u8TRWqVUKfA6aJQsonwi01IPxqA16zX7L4BMa9E4g4DdJW80v3N6jqde4pXeqd2GYCg4AAABJeyJvcmlnaW4iOiJodHRwczovL3dlYnZyLmluZm86NDQzIiwiZmVhdHVyZSI6IldlYlZSIiwiZXhwaXJ5IjoxNDkyMTAzODUyfQ==">
```

Comment:
The date isn't actually used and these are direct copies from the email from the origin trial.

But I agree that the original format is better.  I'll update them.